docs/getting-started/user-guides/using-paymetheus.md

```diff
@@ -0,0 +1,151 @@
+# **The Paymetheus Windows Wallet** #
+This guide assumes you have already set up a Paymetheus wallet using [this guide](paymetheus.md).
+
+---
+
+## **Overview** ##
+The overview tab gives a quick summary of your total DCR (spendable and locked), number of accounts and transactions as well as a list 
+of recent account activity.  
+
+![Overview tab](/img/Paymetheus-overview.png)  
+
+---
+
+
+## **Accounts  ** ##
+The accounts tab shows you the accounts in your wallet and allows you to add new ones. Accounts in Decred work just like bank accounts. 
+They allow you to keep seperate records of your DCR. This feature is most 
+useful for those who run businesses and wish to keep seperate accounts for 
+tax records for example. Transferring DCR across accounts will create a 
+transaction on the blockchain.  
+
+![Overview tab](/img/Paymetheus-accounts.png)  
+
+---
+
+
+## **Scripts** ##
+Currently only used for PoS mining in a pool. If you sign up for a pool, 
+you will be asked to import a script. This is where you will add it.
+See the Purchase Tickets section below for more information.  
+
+![Scripts tab](/img/Paymetheus-import-script.png)  
+
+---
+
+
+## **Create Transaction** ##
+This tab is how you send funds to another address. Simply copy the receivers
+address into the text box and type in the amount of Decred you wish to send.
+The estimated fee will be listed as well. You can click the '+' button to 
+send Decred to multiple accounts in the one transaction if you wish.  
```

Comment:
nitpick: you pay to addresses (or even raw scripts, although you don't mention that feature and that's fine for a beginner's guide), not accounts.